WordPress/gutenberg
# [RNMobile] Use tarball instead of git tag for react-native-editor forked dependencies

## Description


Gutenberg Mobile PR: https://github.com/wordpress-mobile/gutenberg-mobile/pull/3982

Used tarball instead of git tag for react-native-editor forked dependencies. 
They are created by running `npm install` and `npm pack` in the forked repository and committing the `.tgz` file.
These `.tgz` files are later referenced in `packages/react-native-editor/package.json` instead of their git tags.

Related to https://github.com/WordPress/gutenberg/issues/33424

Forked dependency PRs (already merged ✅ ):
- https://github.com/wordpress-mobile/react-native-masked-view/pull/5
- https://github.com/wordpress-mobile/react-native-slider/pull/9
- https://github.com/wordpress-mobile/react-native-gesture-handler/pull/7
- https://github.com/wordpress-mobile/react-native-get-random-values/pull/4
- https://github.com/wordpress-mobile/react-native-hsv-color-picker/pull/5
- https://github.com/wordpress-mobile/react-native-keyboard-aware-scroll-view/pull/15
- https://github.com/wordpress-mobile/react-native-linear-gradient/pull/4
- https://github.com/wordpress-mobile/react-native-prompt-android/pull/6
- https://github.com/wordpress-mobile/react-native-reanimated/pull/5
- https://github.com/wordpress-mobile/react-native-safe-area-context/pull/5
- https://github.com/wordpress-mobile/react-native-screens/pull/5
- https://github.com/wordpress-mobile/react-native-svg/pull/15
- https://github.com/wordpress-mobile/react-native-video/pull/13
- https://github.com/wordpress-mobile/react-native-webview/pull/3

## How has this been tested?



- Verify that "React Native E2E Tests" checks pass on https://github.com/WordPress/gutenberg/pull/35108 (this PR contains the same changes but NPM cache is disabled to force a clean dependency installation)
- Verify that the PR checks of the [related GB-mobile PR](https://github.com/wordpress-mobile/gutenberg-mobile/pull/3982) pass.
- Verify that the [Android](https://github.com/wordpress-mobile/WordPress-Android/pull/15362) and [iOS](https://github.com/wordpress-mobile/WordPress-iOS/pull/17198) versions can produce installable builds.

## Screenshots 

N/A

## Types of changes

Bug fix (non-breaking change which fixes an issue)



## Checklist:
